logdna/resource_view.go

```diff
@@ -99,18 +100,16 @@ func buildChannels(emailchannels []interface{}, pagerdutychannels []interface{},
 		url := i["url"].(string)
 
 		headersMap := make(map[string]string)
-		templateMap := make(map[string]string)
 
 		for k, v := range headers {
 			headersMap[k] = v.(string)
 		}
 
-		for k, v := range bodytemplate {
-			templateMap[k] = v.(string)
-		}
+		var bodyTemplate map[string]interface{}
```

Comment:
best to avoid defining variables that differ only by casing `bodytemplate` vs `bodyTemplate`

see my above comment though, I'm not 100% sure we need this